test/fennec/udp_test.exs

```diff
@@ -228,8 +230,12 @@ defmodule Fennec.UDPTest do
   end
 
   defp udp_communicate(udp, client_id, req) do
+    with_mock Fennec.Auth, [:passthrough], [
```

Comment:
`Fennec.Auth.maybe` based on the method and class, runs authentication and authorization. This mock ensures this doesn't happen.